gen/src/generate/ucd/age.rs

```diff
@@ -74,7 +74,7 @@ impl FromStr for AgeData {
                     age_data.insert(
                         char,
                         format!(
-                            "Assigned(UnicodeVersion {{ major: {}, minor: {}, micro: {} }})",
+                            "UnicodeVersion {{ major: {}, minor: {}, micro: {} }}",
```

Comment:
Yeah! Actually, I think we should do the same for all non-named properties. Doing so, we don't have to make the struct fields public at all!